src/ICS20Transfer.sol

```diff
@@ -186,10 +195,11 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
         returns (ICS20Lib.UnwrappedPacketData memory)
     {
         ICS20Lib.PacketDataJSON memory packetData = ICS20Lib.unmarshalJSON(packet.data);
-        ICS20Lib.UnwrappedPacketData memory receivePacketData = ICS20Lib.UnwrappedPacketData({
-            denom: packetData.denom,
+        ICS20Lib.UnwrappedPacketData memory sendPacketData = ICS20Lib.UnwrappedPacketData({
```

Comment:
I'm super confused about these functions and find it hard to read. I'm curious if there could be a large refactor in a follow up to simplify this stuff. Even the name `UnwrappedPacketData` is confusing imo

Reply:
Yeah, I would love that too :/

Reply:
Made an issue for it: https://github.com/srdtrk/solidity-ibc-eureka/issues/36